frontend/taipy-gui/src/components/Taipy/tableUtils.tsx

```diff
@@ -245,7 +246,13 @@ export const getSortByIndex = (cols: Record<string, ColumnDesc>) => (key1: strin
         ? 1
         : cols[key1].index - cols[key2].index;
 
-const formatValue = (val: RowValue, col: ColumnDesc, formatConf: FormatConfig, nanValue?: string): string => {
+const formatValue = (
+    val: RowValue,
+    col: ColumnDesc,
+    formatConf: FormatConfig,
+    nanValue: string | undefined,
+    lineBreak: boolean = true
```

Comment:
Could the ColumnDesc object hold that setting for a given column, just in case we want to allow walking that back?